solutions/usaco-810.mdx

```diff
@@ -161,4 +161,27 @@ public class RestStops {
 
 </JavaSection>
 
+<PySection>
+
+```python
+MX = 100005
+x, c = [0] * MX, [0] * MX
```

Comment:
wait please just use dynamic arrays
it isn't even that hard
just go to the `with` block
```py
x, c = [0] * (whatever), [0] * (whatever)
```

Reply:
bump